src/harness/renderers/gl/gl_renderer.c

```diff
@@ -29,13 +29,16 @@ static br_pixelmap *last_colour_buffer, *last_depth_buffer;
 static int dirty_buffers = 0;
 
 static GLuint current_framebuffer_texture;
-static int generated_current_framebuffer_for_this_frame = 0;
 
 static br_pixelmap *last_colour_buffer, *last_depth_buffer;
 
 static tStored_material* current_material;
 static br_pixelmap* current_shade_table;
 
+// Increment flush_counter during flush, and upload_counter when we upload the framebuffer texture.
+// If the counters are equal, we can avoid re-uploading the same thing.
+static int flush_counter = 0, framebuffer_upload_counter = 0;
```

Comment:
I would make these unsigned. Overflow for signed types is UB, but not for unsigned ones.

Reply:
thanks, done 👍